### PR TITLE
Convert util cruft to enhanced implicit classes

### PR DESCRIPTION
**Pull request by geoffjentry**
Minor change to FileUtil to convert to an enhanced class pattern instead of util functions.

Reviewable now, but hold off on merging until the end of the sprint.

### CODE REVIEW

File: src/main/scala/cromwell/util/FileUtil.scala [hunk 1]
```diff
@@ -27,11 +19,25 @@ object FileUtil {
...
     val source = io.Source.fromFile(file)
     try source.mkString finally source.close()
   }
+
+  implicit class EnhancedFile(val file: File) extends AnyVal {
+    /** Read an entire file into a string, closing the underlying stream. */
+    def slurp: String = {
+      val source = io.Source.fromFile(file)
+      try source.mkString finally source.close()
+    }
```
Comment (mcovarr) [inline src/main/scala/cromwell/util/FileUtil.scala]: Why not call (or remove if no longer used) `FileUtil.slurp`?

Comment (geoffjentry) [inline src/main/scala/cromwell/util/FileUtil.scala]: The whole point was to remove `FileUtil.slurp`

Comment (geoffjentry) [inline src/main/scala/cromwell/util/FileUtil.scala]: Oh, there was a lack of deleting on my part :)

Comment (geoffjentry) [inline src/main/scala/cromwell/util/FileUtil.scala]: It be gone now

**Comment by mcovarr** [comment]
Happy to merge this whenever you like.

**Comment by scottfrazer** [comment]
:+1:

**Pull request merged**